requirements.txt

```diff
@@ -2,4 +2,3 @@ absl-py==0.10.0
 pygerrit2==2.0.13
 requests==2.24.0
 requests-oauthlib==1.3.0
-absl-py
```

Comment:
Hmm, did this get dropped accidentally?
We're still doing a pip install for it.

Also (relevant to a parent commit), if we have a requirements.txt, can we just rely on the `pip install -r` call?

Reply:
It wasn't in the original requirements file. The build in GH Actions was breaking because it was missing that module. So I added it in the requirements file (didn't work), so then I removed it and added it as a pip install in the .yml